app/src/components/ViewEvent/ViewParticipants.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import style from './ViewEventContainer.module.scss';
+import { stringifyEmail } from 'src/types/email';
+import { useParticipants } from 'src/hooks/cache';
+import { hasLoaded } from 'src/remote-data';
+
+interface IProps {
+  eventId: string;
+  editToken?: string;
+}
+
+export const ViewParticipants = ({ eventId, editToken }: IProps) => {
+  const remoteParticipants = useParticipants(eventId, editToken);
+
+  const attendees = hasLoaded(remoteParticipants)
+    ? remoteParticipants.data.attendees
+    : [];
+  const waitingList = hasLoaded(remoteParticipants)
+    ? remoteParticipants.data.waitingList
+    : [];
```

Comment:
Vil vi ha tomme lister dersom ingenting er lasta?
Dersom det ikke er lastet er lista tom og dersom lista er tom vises teksten "ingen påmeldte", men det trenger vell nødvendigvis ikke å være sant? 🤔

Reply:
Ja det er ikke optimalt. Husker du hvorfor det var gjort slik i utgangspunktet @tskj? Den står som grønn fordi jeg la det i egen komponent, men var slik tidligere også. Kanskje endre så det blir tydeligere om den laster eller er tom?

Reply:
Ingen anelse, men er verdt å prøve å skrive på ein slik måte at vi kan skrive "laster" mens vi venter på dataen. Gjelder uanz bare for admin og arrangør.